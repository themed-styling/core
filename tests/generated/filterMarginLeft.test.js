import 'jest-styled-components'

import { filterMarginLeft } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(filterMarginLeft, 'filterMarginLeft')
