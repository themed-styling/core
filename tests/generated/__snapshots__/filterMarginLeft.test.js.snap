// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filterMarginLeft renders 0 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: 0;
  filter-margin-left: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 0;
  filter-margin-left: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 0;
  filter-margin-left: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 0;
  filter-margin-left: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders function 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: hasnot;
  filter-margin-left: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: has;
  filter-margin-left: has;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: hasnot;
  filter-margin-left: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: has;
  filter-margin-left: has;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders invalid theme path 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: invalid.path.1;
  filter-margin-left: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: invalid.path.1;
  filter-margin-left: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: invalid.path.1;
  filter-margin-left: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: invalid.path.1;
  filter-margin-left: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders negative number 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: -54881;
  filter-margin-left: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: -54881;
  filter-margin-left: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: -42365;
  filter-margin-left: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: -42365;
  filter-margin-left: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: July;
    filter-margin-left: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: September;
    filter-margin-left: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: July;
    filter-margin-left: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: September;
    filter-margin-left: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: May;
    filter-margin-left: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: June;
    filter-margin-left: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: April;
    filter-margin-left: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: May;
    filter-margin-left: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: June;
    filter-margin-left: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-filter-margin-left: April;
    filter-margin-left: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    -webkit-filter-margin-left: July;
    filter-margin-left: July;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media January {
  .c0 {
    -webkit-filter-margin-left: September;
    filter-margin-left: September;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

@media June {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    -webkit-filter-margin-left: July;
    filter-margin-left: July;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media January {
  .c0 {
    -webkit-filter-margin-left: September;
    filter-margin-left: September;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: November;
    filter-margin-left: November;
  }
}

@media June {
  .c0 {
    -webkit-filter-margin-left: August;
    filter-margin-left: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-filter-margin-left: January;
    filter-margin-left: January;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: December;
    filter-margin-left: December;
  }
}

@media January {
  .c0 {
    -webkit-filter-margin-left: April;
    filter-margin-left: April;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: May;
    filter-margin-left: May;
  }
}

@media June {
  .c0 {
    -webkit-filter-margin-left: June;
    filter-margin-left: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-filter-margin-left: January;
    filter-margin-left: January;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: December;
    filter-margin-left: December;
  }
}

@media January {
  .c0 {
    -webkit-filter-margin-left: April;
    filter-margin-left: April;
  }
}

@media April {
  .c0 {
    -webkit-filter-margin-left: May;
    filter-margin-left: May;
  }
}

@media June {
  .c0 {
    -webkit-filter-margin-left: June;
    filter-margin-left: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders postitive number 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: 54881;
  filter-margin-left: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 54881;
  filter-margin-left: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 42365;
  filter-margin-left: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: 42365;
  filter-margin-left: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders string 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: July;
  filter-margin-left: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: July;
  filter-margin-left: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: June;
  filter-margin-left: June;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: June;
  filter-margin-left: June;
}

<div
    className="c0"
  />,
]
`;

exports[`filterMarginLeft renders valid theme path 1`] = `
Array [
  .c0 {
  -webkit-filter-margin-left: DarkSlateBlue;
  filter-margin-left: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: DarkSlateBlue;
  filter-margin-left: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: DarkSlateBlue;
  filter-margin-left: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-filter-margin-left: DarkSlateBlue;
  filter-margin-left: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
